contracts/FractalUsul.sol

```diff
@@ -1,86 +1,469 @@
-//SPDX-License-Identifier: MIT
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
"LGPL imposes no conditions on using the code in software that's sold commercially."  So I believe we're good on that front

Reply:
not that we have plans to, though ha